hail_search/web_app.py

```diff
@@ -1,9 +1,29 @@
 from aiohttp import web
 import json
 import hail as hl
+import logging
 
 from hail_search.search import search_hail_backend, load_globals
 
+logger = logging.getLogger(__name__)
+
+
+def _handle_exception(e, request):
+    if not e.text:
```

Comment:
Does this line ever run?  It seems to me like only subclasses of `HTTPError` will enter here, and it seems like `aiohttp` manages the `reason` -> `text` propagation.

Reply:
this line always runs, HTTPError leaves text blank by default

Reply:
I think it [gets set for us](https://docs.aiohttp.org/en/v3.8.0/_modules/aiohttp/web_exceptions.html):
```
In [3]: try:
   ...:      raise HTTPError(reason='hello world')
   ...:   except HTTPError as e:
   ...:      print(e.text)
   ...:
-1: hello world
```